.circleci/continue_config.yml

```diff
@@ -346,11 +340,6 @@ workflows:
           matrix:
             parameters:
               python_version: ['3.7', '3.8', '3.9', '3.10']
-          filters:
```

Comment:
Why are we removing branch filters from win_unit_tests ? I saw that for e2e tests we skip the behave scenario in package/features/environment.py , does that apply to unit tests as well ?

Reply:
ok good question -  so this is just for test purpose. 
win-e2e and win-unit tests are only done when you merge to main, and during demo. it doesn't happen when you are doing commits in your PR. The reason is because win tests take too long 20min or so to execute and it kind of becomes a blocker. 

I have only temporarily removed this because I want to check how e2e test work on windows. I will remove it before I merge.

Reply:
we skip testing for python>=3.9 and kedro 17.5 because kedro 17.5 does not support python > 3.8 , so it's no point testing that scenario